custom/enikshay/ucr/data_sources/voucher_v2.json

```diff
@@ -0,0 +1,1184 @@
+{
+    "domains": [
+        "enikshay",
+        "sheel-enikshay",
+        "enikshay-reports-qa",
+        "enikshay-performance-test"
+    ],
+    "server_environment": [
+        "enikshay",
+        "softlayer"
```

Comment:
we don't need to add new datasources to softlayer any more

Reply:
@proteusvacuum sure will do this in two commits next time.